wdl/ClusterBatch.wdl

```diff
@@ -65,6 +66,8 @@ workflow ClusterBatch {
 
     Float? java_mem_fraction
 
+    Int outlier_cutoff_nIQR
```

Comment:
I'd suggest making this an optional. If it's provided, we run PlotSVCountsPerSample. If not, it skips the task (we don't need to run this for the single sample pipeline for example). Would need to add this input and the outputs to GATKSVPipelinePhase1 and GATKSVPipelineBatch.